cmd/evm/runner.go

```diff
@@ -183,15 +197,30 @@ func runCmd(ctx *cli.Context) error {
 	}
 	tstart := time.Now()
 	var leftOverGas uint64
+
+	// sender account pre-pays gas
+	mgval := new(big.Int).Mul(new(big.Int).SetUint64(initialGas), runtimeConfig.GasPrice)
+	statedb.SubBalance(sender, mgval)
```

Comment:
This will likely throw if the `sender` is not in prestate (?). Would be good if it's still possible to just do `evm --code 6040 run` and not worry about the rest.